docs/en/observability/apm/collect-application-data/k8s-attacher.asciidoc

```diff
@@ -1,6 +1,15 @@
+////
+TO DO:
+Does this need to be its own page?
+////
+
```

Comment:
Similar question for @AlexanderWert here. Do we want this to have it's own page?

38% of users who view this page end up clicking through to the attacher book. However, only 13% of the attacher book views originate from this page.

The other option is to move this to https://observability-docs_bk_4230.docs-preview.app.elstc.co/guide/en/observability/master/apm-collect-application-data.html (just like with AWS lambda)

Reply:
Let's keep this page for now, and we can revisit this question in https://github.com/elastic/obs-docs-projects/issues/269.